Packs/EmailCommunication/Scripts/SendEmailReply/SendEmailReply.py

```diff
@@ -142,22 +142,23 @@ def create_file_data_json(attachment):
 def get_reply_body(notes, incident_id, attachments):
     """ Get the notes and the incident id and return the reply body
     Args:
-        notes (dict): The notes of the email.
+        notes (list): The notes of the email.
         incident_id (str): The incident id.
         attachments (list): The email's attachments.
     Returns:
         The reply body and the html body.
     """
     reply_body = ''
     if notes:
+        if attachments:
+            attachment_names = [attachment.get('name') for attachment in attachments]
+            reply_body += f'Attachments: {attachment_names}\n\n'
```

Comment:
Lets save this as a string and add it to the reply_body where the old attachments were added so we will keep the same order.